sw/device/lib/testing/test_framework/ottf_isrs.S

```diff
@@ -19,11 +19,20 @@
    * b11 (0x3), which means that the trapped instruction is not compressed,
    * i.e., the trapped instruction is 32bits = 4bytes. Otherwise, the trapped
    * instruction is 16bits = 2bytes.
+   *
+   * Before loading the instruction pointed to by the `mepc` we should check
+   * whether the `mepc` address is invalid by checking the `mcause` register for the
+   * `Instruction access fault` code. It is invalid then return `mpec`.
```

Comment:
How does loading an invalid `mepc` in a temporary register cause `mcause` to indicate an instruction access fault?

Reply:
The problem is loading the content of the memory pointed by the `mepc`, when `mepc` contain a invalid address.